Essentials/src/config.yml

```diff
@@ -692,6 +692,16 @@ max-money: 10000000000000
 # Setting this to 0, will disable overdrafts/loans completely.  Users need 'essentials.eco.loan' perm to go below 0.
 min-money: -10000
 
+# Enable this to make sell signs automatically adjust their price according to worth.yml
+economy-update-sell-sign-to-worth: false
+
+# Enable this to make buy signs automatically adjust their price according to worth.yml
+economy-update-buy-sign-to-worth: false
+
+# If you don't want buy and sell prices to be the same, set this multiplier to something higher then 1.0
+# All buy prices for players will be multiplied by this amount.
```

Comment:
This description should be improved. It's not clear if this only happens when the above booleans are enabled.

Reply:
true, I'll fix it